### DISCUSSION TITLE
Is this project still ongoing?

### DISCUSSION THREAD

**Discussion by caraar12345** [Madelena/Metrology-for-Hass#72] [General]
Hello!

Just a quick one - out of interest, is Metrology a project that is still being worked on?

It's been around a year since the last update and I know that Madelena is now the design lead at Nabu Casa (congrats!) so it'd make sense if it were to be 'finished'.

**Comment by Madelena** [comment]
I will continue to update it to fix bugs due to changes in newer versions of Home Assistant, but otherwise the theme is pretty much complete. 😊
